### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by jeevansurendran**
- swipe to refresh for manager invoice
- [Task to fix for small screen sizes](https://app.clickup.com/t/27h3nq)

### CODE REVIEW

**Comment by shivanshs9** [comment]
Why do you name your commits like so? It doesn't give any insight into what your commit actually changed.

**Comment by jeevansurendran** [comment]
I usually combine all the commit to a single commit in the end. So that the commits changes are easier for me to see and so that I can check if I have accidentally named some things wrong or used hard coded dimension values by accident.

File: app/src/main/java/com/checkin/app/checkin/manager/fragments/ManagerInvoiceFragment.kt [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    private fun setupUi() {
+        rvSessions.layoutManager = LinearLayoutManager(requireContext(), RecyclerView.VERTICAL, false)
+        rvSessions.adapter = mAdapter
+    }
+
+    override fun updateScreen() {
+        super.updateScreen()
+        mViewModel.fetchShopSessions(mViewModel.mShopPk)
+        mViewModel.filterFrom(Utils.getCurrentFormattedDateInvoice())
+        mViewModel.filterTo(Utils.getCurrentFormattedDateInvoice())
```
Comment (jeevansurendran) [inline app/src/main/java/com/checkin/app/checkin/manager/fragments/ManagerInvoiceFragment.kt]: Is this the correct way ????

Comment (jeevansurendran) [reply app/src/main/java/com/checkin/app/checkin/manager/fragments/ManagerInvoiceFragment.kt]: for refreshing invoice fragment. Something similar was done in the activity.

File: app/src/main/java/com/checkin/app/checkin/Shop/Private/Invoice/ShopInvoiceViewModel.kt [hunk 2]
```diff
@@ -8,7 +8,7 @@ import com.checkin.app.checkin.data.resource.Resource
 class ShopInvoiceViewModel(application: Application) : BaseViewModel(application) {
     private val mShopInvoiceRepository: ShopInvoiceRepository = ShopInvoiceRepository.getInstance(application)
 
-    private var mShopPk: Long = 0
+    public var mShopPk: Long = 0
```
Comment (shivanshs9) [inline app/src/main/java/com/checkin/app/checkin/Shop/Private/Invoice/ShopInvoiceViewModel.kt]: Convention is to prefix with 's' only private members, not the public members.

Comment (shivanshs9) [reply app/src/main/java/com/checkin/app/checkin/Shop/Private/Invoice/ShopInvoiceViewModel.kt]: You don't even need to make it public for this usecase

File: app/src/main/java/com/checkin/app/checkin/manager/fragments/ManagerInvoiceFragment.kt (hunk 1 above)
Comment (shivanshs9) [reply app/src/main/java/com/checkin/app/checkin/manager/fragments/ManagerInvoiceFragment.kt]: No, there's a **updateResults** method in every viewmodel for a reason. And also, you should apply the filters before refetching data

File: app/src/main/res/layout/activity_manager_preorder_session_detail.xml [hunk 3]
```diff
@@ -35,34 +32,37 @@
...
                 <TextView
                     android:id="@+id/tv_manager_scheduled_session_customer_visits"
                     style="@style/PinkishGreySmallTextStyle"
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
+                    android:layout_marginBottom="@dimen/spacing_extra_small"
                     android:gravity="bottom|start"
                     app:layout_constraintBottom_toBottomOf="parent"
                     app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toBottomOf="@+id/tv_manager_scheduled_session_customer_name"
```
Comment (shivanshs9) [inline app/src/main/res/layout/activity_manager_preorder_session_detail.xml]: You again introduced **@+id** here, even though it's wrong.

File: app/src/main/res/layout/activity_manager_preorder_session_detail.xml [hunk 4]
```diff
@@ -35,34 +32,37 @@
...
-
+    
                 <TextView
                     android:id="@+id/tv_manager_scheduled_call"
                     style="@style/CallButtonTextStyle"
                     app:layout_constraintBottom_toBottomOf="parent"
                     app:layout_constraintEnd_toEndOf="parent"
-                    app:layout_constraintTop_toBottomOf="@id/tv_manager_scheduled_session_order_id"
-                    app:layout_constraintVertical_bias="1" />
+                    app:layout_constraintTop_toBottomOf="@+id/tv_manager_scheduled_session_order_id"
```
Comment (shivanshs9) [inline app/src/main/res/layout/activity_manager_preorder_session_detail.xml]: Same here.

File: app/src/main/res/layout/item_manager_preorder_preparation_table.xml [hunk 5]
```diff
@@ -58,14 +63,14 @@
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintTop_toTopOf="parent"
             tools:text="Now" />
-
+    
         <TextView
             android:id="@+id/tv_manager_preorder_call"
             style="@style/CallButtonTextStyle"
-            android:layout_marginTop="@dimen/spacing_extra_tiny"
+            app:layout_constraintBottom_toBottomOf="@+id/tv_manager_preorder_scheduled_info"
```
Comment (shivanshs9) [inline app/src/main/res/layout/item_manager_preorder_preparation_table.xml]: same here.

**Comment by jeevansurendran** [comment]
made the requested changes

**Comment by jeevansurendran** [comment]
Fix for PaytmPayment

**Comment by shivanshs9** [comment]
Merged via 6ae2a0a57d81e7d318ff499a4c32deb8761caafd

**Pull request closed without merging**